ingest/esimmon/cli/watch/__init__.py

```diff
@@ -417,38 +424,46 @@ async def update_range_metadata(gc, image_tarball, bp_path, variable_items, sema
                 attrs = json.loads(attrs[0])
                 vars = fh.available_variables()
 
-                if attrs["type"] != "mesh-colormap":
-                    # All plots except mesh-colormap have x and y vars to grab
+                if y_names := attrs.get("y", None):
```

Comment:
You might be able to refactor all of those via a loop on ["x", "color", "nodes"]. Not sure why "y" is different.

Reply:
There may be more than one y var being plotted so we need to look at the range for each variable. I could still refactor this for `["x", "color", "nodes"]` though.

Reply:
Simplified the check for `x`, `y`, and `color` attributes